### DISCUSSION TITLE
Change Hotkeys/Key Bindings

### DISCUSSION THREAD

**Discussion by djravine** [djravine/idlecombos#17] [Ideas]
## Change Hotkeys/Key Bindings - POSSIBLE FUTURE
To store edited hotkeys will require a fair amount of work, otherwise they would reset on reload/reopen.
Might look at this in the future.

**Comment by raxele123** [comment]
Would you indulge me and put Tiny Blacksmiths on numpad +, Small Blacksmiths on numpad -, Medium Blacksmiths on numpad *, and Large Blacksmiths on numpad /? Not a big fan of the Alt + 3 different letters depending on what hotkey you're trying to use. Could you add these additional hotkeys? A single press takes you directly to it? Also you might want to put something in idlecombos so people don't have to visit github to find out the hotkeys. Please and thank you! :3

**Reply by djravine** [reply]
These current hotkeys are standard menu hotkeys that have been around since the early versions of windows. They have always been with the ALT key and you can see the underscored letter for each sub menu.

I'll see if I can add extra hotkeys.

**Reply by raxele123** [reply]
Lmao I saw the hotkey changes. Could you flip it? So the contracts are ctrl + num 1(tiny), ctrl + num 2(small), ctrl + num 3(medium, etc. Yes I'm the one who asked for it to be the way you made it currently but with ctrl its a whole different story and its perfectly fine that way too. As per usual please and thank you! :3

**Reply by djravine** [reply]
Updated Hotkeys for you:
* CONTROL + NUMPAD1 - Tiny Blacksmith Contracts
* CONTROL + NUMPAD2 - Small Blacksmith Contracts
* CONTROL + NUMPAD3 - Medium Blacksmith Contracts
* CONTROL + NUMPAD4 - Large Blacksmith Contracts
* CONTROL + NUMPAD5 - Huge Blacksmith Contracts
* CONTROL + NUMPAD/ - Buy Silver Chests
* CONTROL + NUMPAD* - Buy Gold Chests
* CONTROL + NUMPAD- - Buy Event Chests
* CONTROL + NUMPAD7 - Open Silver Chests
* CONTROL + NUMPAD8 - Open Gold Chests
* CONTROL + NUMPAD9 - Open Event Chests

**Comment by djravine** [comment]
Issue https://github.com/djravine/idlecombos/issues/23
PR https://github.com/djravine/idlecombos/pull/29
Released [v3.38](https://github.com/djravine/idlecombos/releases/tag/v3.38)

**Comment by djravine** [comment]
Updated hotkeys

Released [v3.39](https://github.com/djravine/idlecombos/releases/tag/v3.39)
